excel.go

```diff
@@ -224,6 +250,44 @@ func copyCell(f *excelize.File, automatedSheetName string, srcX, srcY, destX, de
 	return nil
 }
 
+func sortRows(f *excelize.File, input *Input, sheetname string) error {
+	rows, err := f.GetRows(sheetname)
+	if err != nil {
+		return err
+	}
+
```

Comment:
I think you could replace lines 259–274, and the new types, with
```
sort.Slice(rows[input.RowOffset:], func(i, j int) bool {
  return rows[input.RowOffset + i][ColUser] < rows[input.RowOffset + j][ColUser]
})
```

Reply:
Yes, much simpler to use `sort.Slice` . I get the col index from the map input.AutomatedSheetColNameToIndex, rather than use `ColUser` directly.

Reply:
> I get the col index from the map input.AutomatedSheetColNameToIndex, rather than use ColUser directly.
Yes, that is the correct thing, but what is currently in the PR is my incorrect suggestion. This is caught by the new test, btw.

Reply:
Yes, back to using `colUser` from the map `input.AutomatedSheetColNameToIndex`.